Triangulation_2/include/CGAL/Triangulation_euclidean_traits_2.h

```diff
@@ -21,6 +21,9 @@
 #ifndef CGAL_TRIANGULATION_EUCLIDEAN_TRAITS_2_H
 #define CGAL_TRIANGULATION_EUCLIDEAN_TRAITS_2_H
 
```

Comment:
You removed the other deprecated header but not this one. Any reason for that choice?